### ISSUE TITLE
Missing Alt text/aria label

### ISSUE THREAD

**Issue by fosterc1** [cgkineo/adapt-devtools#56]
Hello

The Devtools plugin incorrectly reports no alt text/aria-label for hotgrid items, as the hotgrid plugin combines the item title with the graphic alt text into one aria label that appears as a child element of the hotgrid item button.

Thanks
![171817727-4dc59326-00b4-437e-94ae-d08aa2248dbb](https://user-images.githubusercontent.com/29701150/173101276-214007c9-d598-4333-aaf2-e0967380030e.jpeg)
![171817737-23f8e79a-fa97-40b7-94d2-dffeeb56e2ab](https://user-images.githubusercontent.com/29701150/173101285-99bdb7cf-2464-459f-ae95-26b70c961665.jpeg)

**Comment by fosterc1** [comment]
The Devtools plugin also doesnt display the aria label where svgz images have been used, using either of the following plugins: https://github.com/cgkineo/adapt-graphicLottie, https://github.com/cgkineo/adapt-svg

Thanks

**Comment by guywillis** [comment]
Hi @fosterc1,

The Hotgrid item image alt text / aria label works in a slightly different way to the traditional method.

The standard way would be to attach an alt / aria label attribute to the element and then define whether that element has alternative text or not. See the Hotgraphic as an example: https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/master/templates/hotgraphicLayoutPins.jsx#L23

The Hotgrid, however, requires a solution that is a little more complicated as multiple text sources are being used to define the alternative text. See here for the Hotgrid aria label implementation: https://github.com/cgkineo/adapt-hotgrid/blob/master/templates/hotgrid.jsx#L81 and the corresponding function: https://github.com/cgkineo/adapt-hotgrid/blob/8a75a5b405385d98f3e985e036823356268bf6e7/templates/hotgrid.jsx#L20

The DevTools plugin does not currently know this additional element with aria label exists so returns a false for the Hotgrid item images alt text / aria label.

The graphicLottie plugin, I believe, didn't have alt text support at the time this issue was raised but has subsequently been updated. 
https://github.com/cgkineo/adapt-graphicLottie/blob/7361c2122fd351e7449c90849e63f60f0c6a94e6/templates/graphicLottie.jsx#L26

The SVG plugin plugin does have alt text support in a similar fashion to the Hotgrid so it won't show up via DevTools either. 
https://github.com/cgkineo/adapt-svg/blob/df526e3cf0c231aa5aaa671099b7af4a1973a400/templates/svg.hbs#L18
